startup.sh

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
 
 bundle install
-bundle exec rails s -p 3065
+
+if [[ $1 == "--live" ]] ; then
+  GOVUK_APP_DOMAIN=www.gov.uk \
+  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
+  PLEK_SERVICE_CONTENT_STORE_URI=https://www.gov.uk/api \
+  PLEK_SERVICE_STATIC_URI=assets.publishing.service.gov.uk \
+  bundle exec rails s -p 3065
+else
+  bundle exec rails s -p 3065
+fi
```

Comment:
I'd recommend just exporting all the environment variables, and then just having one call to `bundle ...` outside of the if. I think that would be neater.